cmd/clusterctl/client/cluster/components.go

```diff
@@ -209,6 +215,12 @@ func (p *providerComponents) Delete(options DeleteOptions) error {
 			continue
 		}
 
+		if obj.GetKind() == deploymentKind {
+			if err := scaleDownDeployment(ctx, cs, obj); err != nil {
+				return err
```

Comment:
```suggestion
				return errors.Wrap(err, "failed to scale down deployment")
```

Reply:
I added it one layer below as I had to do that anyway